src/context/user-functions/addToHistory.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios"
+
+const addToHistory = (video,userDispatch) => {
+    const encodedToken = localStorage.getItem("userToken")
+    axios
+     .post("/api/user/history",{video},{headers : {authorization : encodedToken}})
+     .then(res => userDispatch({type : "HISTORY" , payload : res.data.history}))
+     .catch(err => console.log(err))
```

Comment:
you can use async await also to make your code more cleaner

Reply:
``

Reply:
I'm more comfortable with promises for now but yes I'll try to get hands on with async-await as well in my next project